cmd/gomod-zip/zip.go

```diff
@@ -133,6 +133,27 @@ func main() {
 	defer dst.Close()
 	zw := zip.NewWriter(dst)
 
+	haveGoMod := make(map[string]bool)
+	for _, zf := range zr.File {
+		dir, file := path.Split(zf.Name)
+		if file == "go.mod" {
```

Comment:
would it be simpler to only track submodules?

```
isSubmodule := map[string]bool{}
for _, zf := range zr.File {
  if zf.Name == prefix + "go.mod" {
    continue
  }
  dir, file := path.Split(zf.Name)
  if file == "go.mod" {
    isSubmodule[dir] = true
  }
}
```

Reply:
Updated :+1: